packages/application-components/src/components/main-pages/custom-form-main-page/custom-form-main-page.tsx

```diff
@@ -70,31 +66,54 @@ const CustomFormMainPage = (props: CustomFormMainPageProps) => {
 
   return (
     <PageWrapper>
-      <CustomViewsSelector
-        customViewLocatorCode={props.customViewLocatorCode}
-      />
       <MainPageContainer>
-        <Spacings.Stack scale="l">
-          {props.customTitleRow || (
-            <PageHeaderTitle
-              title={props.title ?? ''}
-              subtitle={props.subtitle}
-              titleSize="big"
-            />
-          )}
-          {!props.hideControls && props.formControls && (
+        {props.customTitleRow || (
+          <PageHeaderTitle
+            title={props.title ?? ''}
+            subtitle={props.subtitle}
+            titleSize="big"
+          />
+        )}
+        <CustomViewsSelector
+          margin={headerRowMargin}
+          customViewLocatorCode={props.customViewLocatorCode}
+        />
+        {!props.hideControls && props.formControls && (
+          <div
+            css={css`
+              margin: ${headerRowMargin};
+            `}
+          >
             <Spacings.Inline justifyContent="flex-end">
               {props.formControls}
             </Spacings.Inline>
-          )}
-          {!props.hideDivider && <Divider />}
-        </Spacings.Stack>
+          </div>
+        )}
+        {!props.hideDivider && (
+          <div
+            css={css`
+              & > hr {
+                margin: ${headerRowMargin};
+              }
```

Comment:
With Emotion it should be possible to [target another Emotion component](https://emotion.sh/docs/styled#targeting-another-emotion-component)

```suggestion
              ${Divider} {
                margin: ${headerRowMargin};
              }
```

Reply:
Thanks for the tip 👍 

Updated here: 69a8b4192cdf1a77c4351dcbd8ea8e57aca35b81